react/NestedSelect/NestedSelect.jsx

```diff
@@ -86,13 +87,28 @@ class NestedSelect extends Component {
       transformParentItem,
       radioPosition
     } = this.props
-    const { history } = this.state
+    const { history, searchValue, searchResult } = this.state
     const current = history[0]
     const children = current.children || []
     const level = history.length - 1
     const isSelectedWithLevel = item => isSelected(item, level)
     const parentItem = transformParentItem(omit(current, 'children'))
 
+    const searchOptions = this.props.searchOptions
+    const hasSearchResult =
+      searchOptions &&
+      searchOptions.displaySearchResultItem &&
```

Comment:
hasSearchResult shouldn't be coupled to displaySearchResultItem. 

We can have searchResults without wanting to display them

Reply:
> Ca n'a pas de rapport avec ta PR mais dans l'exemple quand on sélectionne `E` ça sélectionne `A` également. Je ne sais pas si t'as le temps d'en profiter pour regarder ? (sinon issue...)

J'ai l'impression que c'est voulu (voir isParent // Crude parent-children relationship)
@Crash-- une info la dessus ?